tasks/mpi/example/perf_tests/main.cpp

```diff
@@ -78,8 +78,8 @@ TEST(mpi_example_perf_test, test_task_run) {
   auto perfResults = std::make_shared<ppc::core::PerfResults>();
 
   // Create Perf analyzer
-  auto perfAnalyzer = std::make_shared<ppc::core::Perf>(testMpiTaskParallel);
```

Comment:
This is almost the same as allocating a shared_ptr to PerfAnalyzer - it is freed anyway after the test is completed and is not passed anywhere, why not place it on the stack?